demos/nbody/sim.hpp

```diff
@@ -95,7 +97,7 @@ enum class integrator_t {
 };
 
 template <typename num_t>
-class GravSim {
+class GravSim {  
```

Comment:
Some extra whitespace, are the files formatted?

Reply:
Yes, this is supposed to be fixed already